image/cli/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM ubuntu
```

Comment:
This file is incomplete. I need to add back the entrypoint, which sets the env vars.